www/webapp/src/assets/global.css

```diff
@@ -0,0 +1,5 @@
+/* Dummy for a blank icon. Used to align input fields. */
+.mdi-blank::before {
+    content: "\F68C";
```

Comment:
Why this one specifically?

If I'm understanding this correctly, this is in a private-use area of the codepoint space, so it's rendering (in particular, width), is undefined. Why should one expect this to work?